steamtinkerlaunch

```diff
@@ -7475,8 +7493,8 @@ function prepareReshadeFiles {
 		dld3d47 "32"
 		dld3d47 "64"
 
-		if [ ! -f "$RESHADESRCDIR/$RS_64" ] || [ ! -f "$RESHADESRCDIR/$RS_32" ]; then
-			writelog "404" "${FUNCNAME[0]} - '$RESHADESRCDIR/$RS_64' and/or '$RS_32' missing - downloading"
+		if [ ! -f "$RESHADESRCDIR/$RSVERS/$RS_64" ] || [ ! -f "$RESHADESRCDIR/$RSVERS/$RS_32" ]; then
+			writelog "404" "${FUNCNAME[0]} - '$RESHADESRCDIR/$RSVERS/$RS_64' and/or '$RS_32' missing - downloading"
```

Comment:
I'm not sure if you can install shade for Vulkan games on Linux. if you look frostworx didn't even add an install case for Vulkan games. the JSON is extracted but never really even used so not sure if really nessicary I could add it though since it shouldn't be to hard to do

Reply:
> the JSON is extracted but never really even used so not sure if really nessicary I could add it though since it shouldn't be to hard to do

This is interesting. I haven't tested though I own a couple of games and may test, because we could add a note about this to the wiki. I could see why it might not get used though, I think DXVK just passes through to system Vulkan.

I think it would be nice to have though, in case it ever works in future :-)

Reply:
added skel support for Vulkan only thing missing is installing to the game directory